### PR TITLE
Reformat documentation.

### PR DESCRIPTION
**Pull request by xingweitian**
Hello, this PR tries to fix some markdown syntax errors and format the documentation to make it more readable.

### CODE REVIEW

File: README_2nd_integration.md [hunk 1]
```diff
@@ -1,40 +1,31 @@
 # Tool name: Ontology Type Inference (OTI)
 
----
+## General description
 
-# General description
+OTI supports other code analysis tools by propagating the ground truths about ontic types in the corpus. For example, ontic types help to distinguish two functions with the same Java types and control flow graphs. OTI takes a minimal set of ground truths from other tools and then propagates them properly in the corpus by type inference based on ontology type rules.
```
Comment (wmdietl) [inline README_2nd_integration.md]: For version control it is better to have line breaks.
If you edit a character in the line, the diff will need to store the whole line, which makes it hard to see the change.
With many separate lines, you only have to look at a short line to see the changes.

**Comment by xingweitian** [comment]
@wmdietl Thanks for your advise! I just updated the document, could you please have another look?

**Comment by wmdietl** [comment]
Thanks, this looks good!
One thing for other PRs: You should work in a branch of your fork of the project, not on the master branch. You should reset your master branch to be in sync with this master.

**Comment by xingweitian** [comment]
> Thanks, this looks good!
> One thing for other PRs: You should work in a branch of your fork of the project, not on the master branch. You should reset your master branch to be in sync with this master.

Thanks for your advice! I will do what you suggested.

**Pull request merged**